content/docs/overview/about.md

```diff
@@ -0,0 +1,95 @@
+description: ""
+summary: ""
+date: 2024-08-14T13:52:09+03:00
+lastmod: 2024-08-14T13:52:09+03:00
+draft: false
+weight: 15
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+NuoDB Control Plane (CP) enables customers to consume NuoDB in a Database as a Service (DBaaS) model.
+It runs inside a Kubernetes cluster and handles provisioning and management of multiple domains, and their databases, in the _same_ cluster.
+NuoDB databases are created on-demand remotely using REST services using various predefined configuration options.
+
+## Run on the Cloud
+
+{{< card-grid >}}
+{{< link-card title="Install NuoDB Control Plane" href="../getting-started/installation.md" icon="outline/stack-push" description="Start managing NuoDB databases deployed in Kubernetes" >}}
+{{< link-card title="Deploy NuoDB databases" href="../getting-started/create-database.md" icon="outline/rocket" description="Create NuoDB databases on-demand easily" >}}
+{{< link-card title="Start Developing" href="../getting-started/connect-database.md" icon="outline/code" description="Connect to your database and start developing with NuoDB" >}}
+{{< /card-grid >}}
+
+## Architecture
+
+The NuoDB Control Plane allows the creation of:
+
+- One or more organizations
+- Each organization can have one or more projects.
+Each project corresponds to a NuoDB
+domain with its own set of NuoDB Admin processes (APs).
+- Within a project you can deploy one or more databases.
+When a project is created a new or existing organization must be specified.
+
+NuoDB CP consists of multiple modules that can be deployed independently as part of a microservices-style architecture.
+Each module implements certain aspects of managing domains and databases on behalf of customers.
+The following components are available to support DBaaS using this approach:
+
+- A DBaaS REST Service that exposes multi-tenant access to external customers according to access control rules.
+- A DBaaS Operator that is responsible for enforcing the desired state for databases and domains.
+
+NuoDB domain and database are modeled as Kubernetes [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources) (CRs). Their [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs) act as contracts between NuoDB Operator and the other components.
+
+{{< picture src="overview.png" alt="NuoDB Control Plane overview" >}}
+
+## Key concepts
+
+The NuoDB Control Plane can be broken down into several communicating layers.
+
+{{< picture src="cp-arch-layers.png" alt="NuoDB Control Plane logical architecture" >}}
+
+### REST Service
+
+DBaaS REST Service allows multi-tenant access for external customers according to access control rules.
```

Comment:
Suggested rework: 

DBaaS REST Service facilitates multi-tenant access for external customers in accordance with access control rules. It exposes a REST API, enabling remote management of NuoDB domains and databases through coarse-grained CRUD operations (Create, Read, Update, and Delete). The data model used by the DBaaS REST Service to define user access control is outlined below:

- Organizations consist of multiple users and policies.
- Projects serve as logical groupings of databases, each associated with a specific service tier.
- Users are assigned roles that dictate access to projects and databases.

Projects abstract the relevant aspects of domains for DBaaS users and provide isolation between databases. Databases within the same project will inherit the service tier from the project they belong to, while databases in different projects will be managed in separate domains.

Reply:
I've updated most of the text as per your suggestion.